openmdao/utils/tests/test_units.py

```diff
@@ -245,11 +245,11 @@ def test_name(self):
         y = x2 / (x1**2)
         self.assertEqual(y.name(), 'kg/m**2')
 
-    def test_get_conversion(self):
```

Comment:
I think typically we would add a test for the deprecation as well

Reply:
I can add it, but get_conversion now only does the deprecation warning and returns a call to unit_coversion with the same args, so there's not much being tested

Reply:
yeah, the test would really just be that you get the deprecation warning